tests/installation/system_role.pm

```diff
@@ -7,15 +7,14 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Check system role selection screen or select system role. Added in SLE 12 SP2
-# Maintainer: Jozef Pupava <jpupava@suse.com>
+# Summary: Check default system role selection screen (only for SLE) and select system role. Added in SLE 12 SP2
+# Maintainer: Jozef Pupava <jpupava@suse.com>, Joaquín Rivera <jeriveramoya@suse.com>
```

Comment:
People can assign it to me, but I will most probably not work on it in reasonable time due to another priorities

Reply:
I also don't feel that it's good approach to assign tickets. I see it rather like a point of contact in case you don't know what to do, as not only maintainers introduce changes to the module. Same happens to me, when there is change in a product, etc. and I get the ticket, but it doesn't mean that I will wok on it.